scripts/inactive_study.py

```diff
@@ -64,14 +67,23 @@
                        "for an OpenMC model divided into a number of layers")
 ap = ArgumentParser(description=program_description)
 
-ap.add_argument('-i', dest='script_name', type=str,
-                help='Name of the OpenMC python script to run (without .py extension)')
 ap.add_argument('-n-threads', dest='n_threads', type=int,
                 default=multiprocessing.cpu_count(), help='Number of threads to run Cardinal with')
-ap.add_argument('-input', dest='input_file', type=str,
+ap.add_argument('-i', dest='script_name', type=str, required=True,
+                help='Name of the OpenMC python script to run (without .py extension)')
+ap.add_argument('-input', dest='input_file', type=str, required=True,
                 help='Name of the Cardinal input file to run')
-
+ap.add_argument('--method', dest = 'method', choices =['all','half','window'], required=True,
+                help = "The type of way to detect steady state. Options are all, half, or window")
+opts, rem_args = ap.parse_known_args()
+if(opts.method == "window"):
+    ap.add_argument('--window_length', dest = 'window_length',required = True, type = int,
+                    help =" The window length must be specified if window is the detectio method")
```

Comment:
detectio -> detection

Reply:
Please also change the documentation string here to document what this parameter is, not necessarily the conditions when you need it.